app.js

```diff
@@ -0,0 +1,75 @@
+const addBtn = document.getElementById('add-btn');
+const bookListUi = document.querySelector('.book-list-ui');
+let arrayBooks = [];
+
+// Create a render function
+const renderBook = () => {
+  // Create the book list
+  if (arrayBooks.length === 0) {
+    bookListUi.classList.remove('visible');
+    return;
+  }
+  bookListUi.classList.add('visible');
+
+  bookListUi.innerHTML = '';
+
+  arrayBooks.forEach((book, index) => {
+    const listUi = document.createElement('ul');
+    listUi.setAttribute('class', `list-ui ${index}`);
+
+    const removeBtn = document.createElement('button');
+    removeBtn.setAttribute('class', `remove-btn r-btn${index}`);
+    removeBtn.setAttribute('onclick', `removeBook(${index})`);
+    removeBtn.textContent = 'Remove';
+    /* eslint-disable */
+    for (const key in book) {
+      const bookListElement = document.createElement('li');
+      bookListElement.textContent = book[key];
+      listUi.appendChild(bookListElement);
+    }
+
+    listUi.appendChild(removeBtn);
+    bookListUi.appendChild(listUi);
+  });
+};
+
+function setLocalStorage() {
+  localStorage.setItem('BookList', JSON.stringify(arrayBooks));
+}
+
+// Create the Books array
+const addBookHandler = () => {
```

Comment:
Please ensure that the app performs as expected, Please find below to see the error received when I tried to add a book.
Kindly fix it:
![image](https://user-images.githubusercontent.com/10332499/159470155-f1741957-0459-40b6-ac48-7eac9f1180f7.png)

As a result, I am unable to check the remove book function

Reply:
@micheaol the app works perfectly with the live server could you please give me any help about this issue?

Reply:
@MohammedElseddik ,

I will suggest that you should try to use `condole.log` before pushing. With this you will know the result of the object

Reply:
@micheaol  could you please check it again for me I think I fix it before ask for another review?

Reply:
@MohammedElseddik ,

I still received the same error message, maybe you should take your time to check online for a solution and I will do the same to look up a solution if I can find any.

Reply:
@micheaol  Thank you so much I fix it you can check it it is on line 70  app.js file